docs/changelog.md

```diff
@@ -5,6 +5,30 @@ Lines for version numbers should always be formatted as
 with nothing else on the line.
 -->
 * HEAD
+* [0.32.0](https://pypi.org/project/sematic/0.32.0/)
+    * [feature] Publish a new version of the Dashboard UI, which is currently in "Beta". You can
+      switch to the new version by clicking on the pop-up banner, and between the two versions
+      through your profile window pop-up.
+    * [improvement] Add constraints to the DB schema in order to improve validations. In case you
+      get any errors during the upgrade, please contact us on
+      [Discord](https://discord.gg/4KZJ6kYVax) so that we can assist you.
+    * [improvement] Added documentation for custom user metrics[^1]
+    * [improvement] Improve error messaging for unschedulable pipeline runs
+    * [improvement] When running the CLI via Bazel, use the current directory as the working
+      directory
+    * [improvement] Minor CI and documentation improvements, fixes, and cleanups
```

Comment:
This doesn't impact users directly, I'd remove it

Reply:
Done.